.lando.yml

```diff
@@ -0,0 +1,75 @@
+name: simplytest-me
+recipe: drupal9
+
+config:
+  php: "7.3"
+  webroot: web
+  xdebug: false
+  database: mariadb:10.3
+
+events:
+  # Clear cache after a database import
+  post-db-import:
+    - appserver: cd $LANDO_WEBROOT && drush cache-rebuild --yes
+
+services:
+  appserver:
+    run:
+      - cd $LANDO_MOUNT && composer install --no-scripts
+
+  nodejs:
+    type: node:14
+    globals:
+      gulp-cli: latest
+    run:
+      - cd $LANDO_MOUNT/web/profiles/contrib/simplytest/themes/simplytest_theme && npm install
+      - cd $LANDO_MOUNT/web/profiles/contrib/simplytest/themes/simplytest_theme && npm run build
+
+  phpmyadmin:
+    type: phpmyadmin
+
+tooling:
+  drush:
+    service: appserver
+    env:
+      DRUSH_OPTIONS_URI: "https://simplytest-me.lndo.site/"
+  compile:
+    service: nodejs
+    cmd: cd $LANDO_MOUNT/web/profiles/contrib/simplytest/themes/simplytest_theme && npm run build
+  gulp:
+    service: nodejs
+  node:
+    service: nodejs
+  npm:
+    service: nodejs
+  phpunit:
+    service: appserver
+    cmd: vendor/bin/phpunit
+  phpcs:
+    service: appserver
+    cmd: vendor/bin/phpcs
+  security-check:
+    service: appserver
+    cmd: "php vendor/sensiolabs/security-checker/security-checker security:check composer.lock"
```

Comment:
This package isn't added. Isn't this service deprecated?

Reply:
Get rid of this also.

Reply:
I will do that.